GitUI/CommandsDialogs/FormCommit.cs

```diff
@@ -102,6 +104,11 @@ public sealed partial class FormCommit : GitModuleForm
         private readonly TranslationString _stageFiles = new TranslationString("Stage {0} files");
         private readonly TranslationString _selectOnlyOneFile = new TranslationString("You must have only one file selected.");
 
+        private readonly TranslationString _stageAll = new TranslationString("Stage All");
+        private readonly TranslationString _stageFiltered = new TranslationString("Stage Filtered");
+        private readonly TranslationString _unstageAll = new TranslationString("Unstage All");
+        private readonly TranslationString _unstageFiltered = new TranslationString("Unstage Filtered");
```

Comment:
```suggestion
        private readonly TranslationString _stageAll = new TranslationString("Stage all");
        private readonly TranslationString _stageFiltered = new TranslationString("Stage filtered");
        private readonly TranslationString _unstageAll = new TranslationString("Unstage all");
        private readonly TranslationString _unstageFiltered = new TranslationString("Unstage filtered");
```

Reply:
Thx @RussKie ! In addition I'd fix this in _FormCommit.Designer_ as well
(https://github.com/gitextensions/gitextensions/blob/master/GitUI/CommandsDialogs/FormCommit.Designer.cs) :
            this.toolStageAllItem.Text = "Stage **a**ll";
or 
            this.toolStageAllItem.Text = **_stageAll.Text**;

...or leave it unchanged and do it in the **constructor** of _FormCommit_ ?
toolStageAllItem.ToolTipText = _stageAll.Text;

Obviously, the same for _unstageFiltered.

Reply:
> this.toolStageAllItem.Text = **_stageAll.Text**;

yes

Reply:
Ouch, that'll break the VS form designer and throw an error "The designer loader did not provide a root component but has not indicated why." 

So I change instead as follows:
_FormCommit.Designer.cs:_ this.toolStageAllItem.Text = "Stage **a**ll";
_FormCommit.cs (new line in the constructor):_ toolStageAllItem.ToolTipText = _stageAll.Text;

Reply:
You mustn't change the designer file manually, as it either likely be undone by the designer itself, or will break the designer.
The strings in the designer do not really make any difference as at runtime we'll load the translated strings anyway.